### PR TITLE
Update list of operation result codes to conform to CAP-38

### LINKED ISSUE #601: List of operations is missing errors for some operations
Some operations are missing documentation for certain errors (for example, change trust). We should review the full list and make sure it is in sync with the XDR.

### What did you do?
Visited https://developers.stellar.org/docs/start/list-of-operations/#change-trust.

### What did you expect to see?
Complete list of error codes.

### What did you see instead?
Missing errors related to CAP-38.

### PR DESCRIPTION
**Pull request by Shaptic**
cc @jonjove, as this should close #601
related: stellar/go#3884

### CODE REVIEW

File: content/docs/start/list-of-operations.mdx [hunk 1]
```diff
@@ -379,6 +379,9 @@ Possible errors:
 | CHANGE_TRUST_INVALID_LIMIT | -3 | The `limit` is not sufficient to hold the current balance of the trustline and still satisfy its buying liabilities. This error occurs when attempting to remove a trustline with a non-zero asset balance. |
 | CHANGE_TRUST_LOW_RESERVE | -4 | This account does not have enough XLM to satisfy the minimum XLM reserve increase caused by adding a subentry and still satisfy its XLM selling liabilities. For every new trustline added to an account, the minimum reserve of XLM that account must hold increases. |
 | CHANGE_TRUST_SELF_NOT_ALLOWED | -5 | The source account attempted to create a trustline for itself, which is not allowed. |
+| CHANGE_TRUST_TRUST_LINE_MISSING | -6 | The asset trustline is missing for the liquidity pool |
+| CHANGE_TRUST_CANNOT_DELETE | -7 | The asset trustline is still referenced by a liquidity pool |
+| CHANGE_TRUST_NOT_AUTH_MAINTAIN_LIABILITIES | -8 | The asset trustline is deauthorized |
```
Comment (ire-and-curses) [inline content/docs/start/list-of-operations.mdx:384]: Missing period at end of sentences (x3)

File: content/docs/start/list-of-operations.mdx [hunk 2]
```diff
@@ -610,6 +614,7 @@ Possible errors:
 | REVOKE_SPONSORSHIP_NOT_SPONSOR | -2 | If the ledgerEntry/signer is sponsored, then the source account must be the sponsor. If the ledgerEntry/signer is not sponsored, the source account must be the owner. This error will be thrown otherwise. |
 | REVOKE_SPONSORSHIP_LOW_RESERVE | -3 | The sponsored account does not have enough XLM to satisfy the minimum balance increase caused by revoking sponsorship on a ledgerEntry/signer it owns, or the sponsor of the source account doesn't have enough XLM to satisfy the minimum balance increase caused by sponsoring a transferred ledgerEntry/signer. |
 | REVOKE_SPONSORSHIP_ONLY_TRANSFERABLE | -4 | Sponsorship cannot be removed from this ledgerEntry. This error will happen if the user tries to remove the sponsorship from a ClaimableBalanceEntry. |
+| REVOKE_SPONSORSHIP_MALFORMED | -5 | One or more of the parameters to the operation were invalid. |
```
Comment (ire-and-curses) [inline content/docs/start/list-of-operations.mdx:617]:
```suggestion
| REVOKE_SPONSORSHIP_MALFORMED | -5 | One or more of the inputs to the operation was malformed. |
```

File: content/docs/start/list-of-operations.mdx [hunk 3]
```diff
@@ -749,8 +755,8 @@ Possible errors:
 
 | Error | Code | Description |
 | --- | --- | --- |
-| LIQUIDITY_POOL_WITHDRAW_MALFORMED | -1 | Bad input. |
-| LIQUIDITY_POOL_WITHDRAW_NO_TRUST | -2 | No trust line for one of the assets. |
+| LIQUIDITY_POOL_WITHDRAW_MALFORMED | -1 | One or more of the inputs to the operation was malformed. |
+| LIQUIDITY_POOL_WITHDRAW_NO_TRUST | -2 | There's no trustline for one of the assets. |
```
Comment (ire-and-curses) [inline content/docs/start/list-of-operations.mdx:759]:
```suggestion
| LIQUIDITY_POOL_WITHDRAW_NO_TRUST | -2 | There is no trustline for one of the assets. |
```

File: content/docs/start/list-of-operations.mdx [hunk 4]
```diff
@@ -716,12 +722,12 @@ Possible errors:
...
-| LIQUIDITY_POOL_DEPOSIT_MALFORMED | -1 | Bad input. |
-| LIQUIDITY_POOL_DEPOSIT_NO_TRUST | -2 | No trust line for one of the assets. |
-| LIQUIDITY_POOL_DEPOSIT_NOT_AUTHORIZED | -3 | Not authorized for one of the assets. |
-| LIQUIDITY_POOL_DEPOSIT_UNDERFUNDED | -4 | Insufficient balance for one of the assets. |
-| LIQUIDITY_POOL_DEPOSIT_LINE_FULL | -5 | The liquidity pool share trust line doesn't have a sufficient limit. |
-| LIQUIDITY_POOL_DEPOSIT_BAD_PRICE | -6 | The deposit price is outside the bounds. |
+| LIQUIDITY_POOL_DEPOSIT_MALFORMED | -1 | One or more of the inputs to the operation was malformed. |
+| LIQUIDITY_POOL_DEPOSIT_NO_TRUST | -2 | No trustline exists for one of the assets being deposited. |
+| LIQUIDITY_POOL_DEPOSIT_NOT_AUTHORIZED | -3 | The account does not have authorization for one of the assets. |
+| LIQUIDITY_POOL_DEPOSIT_UNDERFUNDED | -4 | There's not enough balance of one of the assets to perform the deposit. |
```
Comment (ire-and-curses) [inline content/docs/start/list-of-operations.mdx:728]:
```suggestion
| LIQUIDITY_POOL_DEPOSIT_UNDERFUNDED | -4 | There is not enough balance of one of the assets to perform the deposit. |
```

**Review by ire-and-curses — APPROVED** [review]

File: content/docs/start/list-of-operations.mdx (hunk 1 above)
Comment (Shaptic) [reply content/docs/start/list-of-operations.mdx]: Nice catch, fixed

**Pull request merged**